src/Message/ImportCommandCreator.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Message;
+
+use App\MessageHandler\BitbucketImportCommandHandler;
+use App\MessageHandler\GithubImportCommandHandler;
+
+class ImportCommandCreator {
+    public function create($username, $provider) {
+        if ($provider == 'github') {
+            return new GithubImportCommand($username);
+        }
+        if($provider == 'bitbucket') {
+            return new BitbucketImportCommand($username);
+        }
+    }

```

Comment:
now we can throw exception if there is no provider 
```suggestion
        throw new ProviderNotFound();
    }

```
and then we can catch this exception in the ImportCommand

Reply:
Fixed